.github/workflows/linters.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   rubocop:
     name: Rubocop
     runs-on: ubuntu-22.04
-
+    
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍